internal/precache/workload/pullImages.go

```diff
@@ -74,7 +84,11 @@ func pullImage(image, authFile string, progress *precache.Progress) error {
 			log.Infof("Successfully pulled image: %s", image)
 			break
 		} else {
-			log.Infof("Attempt %d/%d: Failed to pull %s: %v", i+1, MaxRetries, image, err)
+			message := fmt.Sprintf("%v", err)
+			log.Infof("Attempt %d/%d: Failed to pull %s: %s", i+1, MaxRetries, image, message)
+			if strings.Contains(message, "manifest unknown") {
+				break
```

Comment:
Might be good to have a comment here for a future dev, to explain why we're breaking vs a retry